vembrane/ann_types.py

```diff
@@ -210,6 +211,39 @@ def typefunc(x):
         super().__init__(name=name, typefunc=typefunc, **kwargs)
 
 
+class RangeTotal(object):
+    def __init__(self, r, total):
+        self.range = r
+        self.total = total
+
+    @classmethod
+    def from_vep_string(cls, value: str) -> "RangeTotal":
+        v = value.split("/")
+        r = [int(s) for s in v[0].split("-")]
+        if len(r) == 1:
+            return cls(range(r[0], r[0] + 1), int(v[1]))
+        elif len(r) == 2:
+            return cls(range(r[0], r[1] + 1), int(v[1]))
+        else:
+            raise UnexpectedException(
```

Comment:
`UnexpectedException` is a bit unspecific and ambiguous (and this error is not related to doctests anyways!).
It's rather a value (or parse) error I think. Not sure what to call that exactly, perhaps just use ValueError.

Reply:
Yeah, I just took something from the IDE suggestions that sounded good. Thanks for the extra expertise sprinkled on top! :D